spec/rubocop/cop/lint/nested_percent_literal_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+describe RuboCop::Cop::Lint::NestedPercentLiteral do
+  subject(:cop) { described_class.new }
+
+  let(:message) { 'Do not nest percent literals.' }
+
+  def expect_offense(source)
+    inspect_source(source)
+
+    expect(cop.offenses.map(&:message)).to eq([message])
+    expect(cop.highlights).to eq([source])
+  end
+
+  context 'when using percent literals' do
```

Comment:
This context seems rather useless to me.

Reply:
Removed (next commit).